controller/bridge.py

```diff
@@ -39,13 +37,15 @@ def start(version):
         # normalize path to be relative to this folder, not pwd
         path = os.path.join(os.path.dirname(__file__), "../trezord-go/bin")
 
-        command = path + "/trezord-go-v" + version + " -ed 21324:21325 -u=false"
+        command = f"{path}/trezord-go-v{version} -ed 21324:21325 -u=false"
+        print("command", command)
 
         proc = Popen(command, shell=True, preexec_fn=os.setsid)
-        # TODO: - add else condition and check if trezord is running and if i own this process (trezord pid is the same with proc pid)
+        # TODO: - add else condition and check if trezord is running and if i own this process
+        #   (trezord pid is the same with proc pid)
```

Comment:
Currently when bridge is already running, it will not spawn a second one, but will not tell anybody about the situation. It can throw a RuntimeError, so the user can be notified. That would be very similar to assert statements we have elsewhere (assert already not running when trying to run)

What I found as a possible issue while developing is that trezord is running on my PC (NIxOS) by default, so I always need to kill it before starting to run these bridges. Also, killing that requires sudo, so it would be impossible (I think) to just kill it here via python script (unless also run with sudo)

Reply:
I agree. Have I seen correctly that you have added a warning that informs the user that they should kill Bridge first?

Reply:
Correct, in one of the latest commits. It is supposed to solve this issue, that there is another bridge already running.

However, I think this should not happen to regular users running this app in docker container, as it is disconnected from their machines (but I am not completely sure)